Sources/TuistCache/ContentHashing/ContentHasher.swift

```diff
@@ -51,7 +51,11 @@ public final class ContentHasher: ContentHashing {
         return try hash(dictString)
     }
 
-    public func hash(fileAtPath filePath: AbsolutePath) throws -> String {
+    public func hash(path filePath: AbsolutePath) throws -> String {
+        if fileHandler.isFolder(filePath) {
+            let paths = try fileHandler.contentsOfDirectory(filePath)
```

Comment:
You might want to sort those to make sure we hash them in a consistent order.

Reply:
`contentsOfDirectory` seem to return files in the same order, but I can sort them to be sure